### ISSUE TITLE
Didn’t work always got an Error

### ISSUE THREAD

**Issue by clipse2004** [mdaskalov/homebridge-dreambox#2]
Hello,
I tried your Plugin and i can add my Dreambox but I always got these error:

homebridge[4370]: [26.2.2020, 19:45:06] [Dreambox] Error: Unexpected close tag
Feb 26 19:45:06 homebridge-Pi4 homebridge[4370]: Line: 53
Feb 26 19:45:06 homebridge-Pi4 homebridge[4370]: Column: 7
Feb 26 19:45:06 homebridge-Pi4 homebridge[4370]: Char: >
Feb 26 19:45:06 homebridge-Pi4 homebridge[4370]:     at error (/usr/local/lib/node_modules/homebridge-dreambox/node_modules/sax/lib/sax.js:651:10)
Feb 26 19:45:06 homebridge-Pi4 homebridge[4370]:     at strictFail (/usr/local/lib/node_modules/homebridge-dreambox/node_modules/sax/lib/sax.js:677:7)
Feb 26 19:45:06 homebridge-Pi4 homebridge[4370]:     at closeTag (/usr/local/lib/node_modules/homebridge-dreambox/node_modules/sax/lib/sax.js:871:9)
Feb 26 19:45:06 homebridge-Pi4 homebridge[4370]:     at SAXParser.write (/usr/local/lib/node_modules/homebridge-dreambox/node_modules/sax/lib/sax.js:1436:13)
Feb 26 19:45:06 homebridge-Pi4 homebridge[4370]:     at Parser.exports.Parser.Parser.parseString (/usr/local/lib/node_modules/homebridge-dreambox/node_modules/xml2js/lib/parser.js:323:31)
Feb 26 19:45:06 homebridge-Pi4 homebridge[4370]:     at Parser.parseString (/usr/local/lib/node_modules/homebridge-dreambox/node_modules/xml2js/lib/parser.js:5:59)
Feb 26 19:45:06 homebridge-Pi4 homebridge[4370]:     at /usr/local/lib/node_modules/homebridge-dreambox/node_modules/xml2js/lib/parser.js:338:24
Feb 26 19:45:06 homebridge-Pi4 homebridge[4370]:     at new Promise (<anonymous>)
Feb 26 19:45:06 homebridge-Pi4 homebridge[4370]:     at Parser.exports.Parser.Parser.parseStringPromise (/usr/local/lib/node_modules/homebridge-dreambox/node_modules/xml2js/lib/parser.js:336:14)
Feb 26 19:45:06 homebridge-Pi4 homebridge[4370]:     at Parser.parseStringPromise (/usr/local/lib/node_modules/homebridge-dreambox/node_modules/xml2js/lib/parser.js:5:59)
Feb 26 19:45:06 homebridge-Pi4 homebridge[4370]:     at Object.exports.parseStringPromise (/usr/local/lib/node_modules/homebridge-dreambox/node_modules/xml2js/lib/parser.js:378:19)
Feb 26 19:45:06 homebridge-Pi4 homebridge[4370]:     at /usr/local/lib/node_modules/homebridge-dreambox/dreambox-accessory.js:113:30
Feb 26 19:45:06 homebridge-Pi4 homebridge[4370]:     at processTicksAndRejections (internal/process/task_queues.js:97:5)

And the Status „now answer“, what did I wrong?

**Comment by mdaskalov** [comment]
My guess is that you have a channel with the '>' char in the name. The XML returned from the receiver cannot be parsed. Try to use different bouquet - you can create one with only few channels to test first. 
Please find out which url was caused the error and try to call it directly to see what's wrong with the answer.

For example below is the URLcalled to read the channels. Replace `{hostname}` and `{bouquet_name}` (default: favourites) with your configured values. Then look at line 53 in the returned XML.

```
curl http://{hostname}/web/getservices?sRef=1:7:1:0:0:0:0:0:0:0:FROM%20BOUQUET%20%22userbouquet.{bouquet_name}.tv%22%20ORDER%20BY%20bouquet
```

**Comment by clipse2004** [comment]
Same error :/ 
IP is correct, Name of Boquette is same like engima. 
Does we need something to configure in Enigma?

**Comment by mdaskalov** [comment]
I was able to reproduce it. The authentication for local clients is enabled on your Dreambox. In this case the web interface returns HTML which cannot be parsed.
This should be fixed in #5 (Release v0.4.0) - I've introduced basic authentication.
You can now disable the authentication or configure your web interface username / password.

**Comment by clipse2004** [comment]
Hey dude, so after changing some things in enigma, now it works :) but i have anonther problem, when the device is off, the homekit shows me sometimes as "not reachable".
Is it better to use it with MQTT? Where does i configure my host for this?

big thanks =)
